src/@context/MarketMetadata/_shaclType.ts

```diff
@@ -0,0 +1,96 @@
+export interface ShaclSchemaField {
+  maxCount: number
+  minCount: number
+  minLength?: number
+  maxLength?: number
+  pattern?: string
+}
+
+export interface ShaclSchema {
+  chainId: ShaclSchemaField
+  credentials: {
+    allow: {
+      type: ShaclSchemaField
+      values: ShaclSchemaField
+    }
+    deny: {
+      type: ShaclSchemaField
+      values: ShaclSchemaField
+    }
+  }
+  id: ShaclSchemaField
+  metadata: {
+    additionalInformation: any
```

Comment:
be slightly more precise as this really shouldn't be e.g. a top-level string. So:

```
addionalInformation: {
  [key: string]: any
}
```